README.md

```diff
@@ -0,0 +1,24 @@
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
```

Comment:
### [__OPTIONAL__]

Remember that your readme file should explain in detail what this project is about on every step, otherwise, no one will know what is going on

> if this is a setup, you should write, stteing up blah blah blah